modules/test/playwright/tests/headless-builder-web/schemas.spec.ts

```diff
@@ -214,53 +324,203 @@ testFeatureFlagsDisabled(
 );
 
 testFeatureFlagsEnabled(
-	'can see whitelisted object definitions on schema creation with feature flag',
+	'can see allowed object definitions on schema creation with feature flag',
 	async ({apiHelpers, applicationPage, headlessBuilderPage}) => {
 		const objectDefinition =
-			await apiHelpers.objectAdmin.postObjectDefinition({
-				active: true,
-				externalReferenceCode: `objectDefinition`,
-				label: {
-					en_US: `objectDefinition`,
-				},
-				name: `ObjectDefinition`,
-				objectFields: [
+			await apiHelpers.objectAdmin.postObjectDefinition(
+				objectDefinitionData
+			);
+
+		const application = await apiHelpers.object.postObjectEntry(
+			applicationData,
+			'headless-builder/applications'
+		);
+
+		await headlessBuilderPage.goto();
+		await headlessBuilderPage.goToEditApplication(application.title);
+		await applicationPage.goToSchemasTab();
+		await applicationPage.addSchemaButton.click();
+		await applicationPage.schemaObjectDefinitionSelector.click();
+
+		expect(
+			(
+				await applicationPage.page.getByRole('menu').allInnerTexts()
+			)[0].split('\n')
+		).toEqual([
+			'APISort',
+			'APIFilter',
+			'APIEndpoint',
+			'APIProperty',
+			'APISchema',
+			'APIApplication',
```

Comment:
Is it possible to check only the following conditions:

* APIApplication, AccountEntry, User and ObjectDefinition are inside the list
* Organization is not inside the list

I am telling this because if in some point, someone includes new Modifiable System Objects, this test will break

Reply:
(The same would apply to all the tests with the same pattern)